### DISCUSSION TITLE
How can I give more padding to the autocomplete documentation?

### DISCUSSION THREAD

**Discussion by mapringg** [LazyVim/LazyVim#418] [Q&A] [answered]
I want to increase the top and bottom padding to prevent the text from pressing up against the box. In addition, a thin border would be preferable.

<img width="985" alt="Screenshot 2566-03-13 at 02 04 20" src="https://user-images.githubusercontent.com/15067237/224566850-4a16cbe4-987c-4c59-98d8-90049a7eb39f.png">

**Comment by folke** [comment, marked as answer]
check the nvim-cmp docs on how to do this

**Reply by mapringg** [reply]
I got it working now. Is it possible to do the same thing to hover and signature helper?

**Reply by folke** [reply]
you can add a border. not sure about the padding. check the noice docs for that. You want to change the hover view
